spring-cassandra/src/test/java/org/baeldung/springcassandra/repository/PersonRepositoryIntegrationTest.java

```diff
@@ -0,0 +1,89 @@
+package org.baeldung.springcassandra.repository;
+
+import com.datastax.driver.core.utils.UUIDs;
+import org.baeldung.springcassandra.config.CassandraContainerInitializer;
+import org.baeldung.springcassandra.model.Person;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ContextConfiguration;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest
+@Testcontainers
+@ContextConfiguration(initializers = PersonRepositoryIntegrationTest.Initializer.class)
+@EnableConfigurationProperties
+public class PersonRepositoryIntegrationTest extends CassandraContainerInitializer {
+
+    @Autowired
+    private PersonRepository personRepository;
+
+    @Test
+    public void givenValidPersonRecord_whenSavingIt_thenDataIsPersisted() {
+        UUID personId = UUIDs.timeBased();
+        final Person newPerson = Person.builder()
```

Comment:
There are two things here I'd avoid.

1. Lombok - it's really not all that
2. `final` in a unit test

In my opinion, this would be simpler if you just had a parameterized constructor:

```java
Person newPerson = new Person(personId, "Luka", "Modric");
```

You may feel that there are some excellent practices that you can get free from Lombok. My experience of it is that it offers simplicity, and ends up making simple things very complex.